plugins/woocommerce-blocks/assets/js/blocks/cart/style.scss

```diff
@@ -106,8 +107,8 @@
 		margin: 0;
 	}
 	.wc-block-components-shipping-rates-control
-	.wc-block-components-radio-control__option {
-		padding: 0 0 0 em($gap-huge);
+		.wc-block-components-radio-control__option {
+		padding-left: em($gap-huge);
```

Comment:
Fixes some broken shipping selector styles in medium (cart frontend) and small (checkout editor) screens